activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -77,7 +77,7 @@ class << self
         def new_client(conn_params)
           PG.connect(conn_params)
         rescue ::PG::Error => error
-          if conn_params && conn_params[:dbname] && error.message.include?(conn_params[:dbname])
+          if conn_params && conn_params[:dbname] && error.message.include?("database \"#{conn_params[:dbname]}\" does not exist")
```

Comment:
@bitfurry Thoughts on including the full message in the `ActiveRecord::NoDatabaseError` instead? Since it appears that the message might be more informative than whatever we replace it with? It also helps us avoid the localization/internalization concern. I'm not too sure what if the locale of the error message depends on the locale of the PostgreSQL server. Regardless, let's send the message to the user.

Reply:
yup, that makes sense to me. Will make the change

The locale of error message depends on the locale of the PostgresSQL server.